packages/big-design/src/components/AccordionGroup/AccordionPanel/AccordionPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import { Text } from '@bigcommerce/big-design';
+import { ExpandLessIcon, ExpandMoreIcon } from '@bigcommerce/big-design-icons';
+import React, { HTMLAttributes, memo } from 'react';
+
+import { useUniqueId } from '../../../hooks';
+import { BoxProps } from '../../Box/Box';
+import { useAccordion } from '../hooks/useAccordion';
+
+import { StyledAccordion, StyledAccordionButton, StyledBox, StyledGridItem } from './styled';
+
+export interface AccordionPanelProps extends HTMLAttributes<HTMLDivElement> {
+  children: React.ReactNode;
+  header: string;
+  isExpanded: boolean;
+  onClick?: () => void;
+  iconLeft?: React.ReactNode;
+  defaultExpanded?: boolean;
+}
+
+export const AccordionPanel: React.FC<AccordionPanelProps & BoxProps> = memo(
+  ({ children, header, iconLeft, defaultExpanded = false, ...props }) => {
+    const { isExpanded, onClick } = useAccordion({ defaultExpanded });
+    const accordionId = useUniqueId('accordion');
+    const accordionItemId = useUniqueId('accordion-item');
+
+    return (
+      <StyledAccordion {...props} isExpanded={isExpanded}>
```

Comment:
Just curious, not sure the requirements of the accordion component in terms of usability, but do we want this component to let the user style the accordion or should we create a private accordion so it can prevent that?

Reply:
Could you elaborate on what you mean by creating a private accordion?

Reply:
Sure, I've seen some components in BD that they are private or I think they are called raw 😅 . This component is rendered by other component(kind of like a public one), and we can decided what props to pass to the raw component.

Example Button component renders a raw button without style props: https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/Button/Button.tsx#L54

Reply:
Good comment, I think we should probably restrict the `style` as well

Reply:
Wondering if I still need to do this if I added @chanceaclark's suggestion below about removing the ability to restyle 
```
  ({ children, header, iconLeft, defaultExpanded = false, className, style, ...props }) => {
  ```

Reply:
I think this is good ^